.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+
+      # Restore cached dependency 
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+      - run:
+          name: Setup Dependencies
+          command: npm install
+      - run:
+          name: Setup Code Climate test-reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+    
+      # Save the dependency cache
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+          paths:
+            - ./node_modules
+      
+      # Run tests and coverage
+      - run:
```

Comment:
This is good work, let's work on PT story [165020157](https://www.pivotaltracker.com/story/show/165020157). 
And let's work on making it build successfully and we will be good to merge it.

Reply:
@me-x-mi Thank you. Now the feature is done.